FractureElasticityVoigt.C

```diff
@@ -276,7 +276,10 @@ bool FractureElasticityVoigt::evalInt (LocalIntegral& elmInt,
       // Degrade the stresses and strain energy isotropically
       dSdE *= Gc;
       sigma *= Gc;
-      myPhi[fe.iGP] = (U *= Gc);
+      U *= Gc;
+      // do not modify buffer during residual evaluation
+      if (m_mode != SIM::RHS_ONLY)
+        myPhi[fe.iGP] = U;
```

Comment:
I tried this in september (I have a local branch with the similar change from 15.09), but did not find it made much difference. But perhaps it is more consistent in any case, so lets add it.

Reply:
trust me, it makes a difference once the other things are correct.